app/jobs/slack_jobs/inviter_job.rb

```diff
@@ -3,14 +3,10 @@ class InviterJob < SlackJobs
     include Sidekiq::Worker
 
     def perform(email)
```

Comment:
Also let's set the max retries like this: 
https://github.com/mperham/sidekiq/wiki/Advanced-Options#workers

details on wait time are here:
https://github.com/mperham/sidekiq/wiki/Error-Handling#automatic-job-retry

I think setting it to 5 retries gets at about 10 minutes from initial try to unique death handler in which we need to notify the mods to read out.

Reply:
I just tested this with 5 retries set, in case you were curious here is the results:
2019-04-23T05:14:28.630184
2019-04-23T05:14:56.946850
2019-04-23T05:16:03.605952
2019-04-23T05:16:54.764033
2019-04-23T05:19:12.275025